<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnhancementSettingsWidget</class>
 <widget class="QWidget" name="EnhancementSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Rendering Enhancements</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Internal Resolution Scale:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="resolutionScale"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Texture Filtering:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="textureFiltering"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="trueColor">
        <property name="text">
         <string>True Color Rendering (24-bit, disables dithering)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="scaledDithering">
        <property name="text">
         <string>Scaled Dithering (scale dither pattern to resolution)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="widescreenHack">
        <property name="text">
         <string>Widescreen Hack (render 3D in display aspect ratio)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="useSoftwareRendererForReadbacks">
        <property name="text">
          <string>Software Renderer Readbacks (run in parallel for VRAM->CPU transfers)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Display Enhancements</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="disableInterlacing">
        <property name="text">
         <string>Disable Interlacing (force progressive render/scan)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="forceNTSCTimings">
        <property name="text">
         <string>Force NTSC Timings (60hz-on-PAL)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="force43For24Bit">
        <property name="text">
         <string>Force 4:3 For 24-Bit Display (disable widescreen for FMVs)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="chromaSmoothingFor24Bit">
        <property name="text">
         <string>Chroma Smoothing For 24-Bit Display (reduce FMV color blockyness)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>PGXP (Precision Geometry Transform Pipeline)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="pgxpEnable">
        <property name="text">
         <string>Geometry Correction</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="pgxpCulling">
        <property name="text">
         <string>Culling Correction</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="pgxpTextureCorrection">
        <property name="text">
         <string>Texture Correction</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="pgxpPreserveProjPrecision">
        <property name="text">
         <string>Preserve Projection Precision</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="pgxpDepthBuffer">
        <property name="text">
         <string>Depth Buffer (Low Compatibility)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="pgxpCPU">
        <property name="text">
         <string>CPU Mode (Very Slow)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
